HYUNSANG_OH/app.js

```diff
@@ -36,20 +37,47 @@ app.get("/ping", (req,res) => {
     res.json({message : "pong"});
 })
 
+app.get("/allPosts", async (req,res) => {
+    const allPost = await appDataSource.query(
```

Comment:
```suggestion
    const posts = await appDataSource.query(
```

allPost라는 네이밍 보다는 그저 posts가 더 적합해 보입니다. allPost라고 작성하면 아무런 필터링 없이 모든 post를 다 가져온 것처럼 보여집니다.

Reply:
네 알겠습니다!